pkg/infra/httpclient/httpclientprovider/tracing_middleware_test.go

```diff
@@ -1,173 +0,0 @@
-package httpclientprovider
```

Comment:
It's probably a good idea to keep these tests if you update them to work with the new tracing